### PR TITLE
Scale-out of Sinopia Cookbook

### PR DESCRIPTION
**Pull request by baccenfutter**
# Change

This PR scales-out our changes to the upstream `Sinopia` cookbook into `ies-sinopia`.

# CR

- please update the stable PR post-merge

Related: easybib/ops#213

### CODE REVIEW

**Comment by baccenfutter** [comment]
@till, @fh: Could you please give this a quick review if everything is going into the right direction, before I start writing tests for what I got.

File: ies-virtualbox/metadata.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+name              'ies-virtualbox'
+maintainer        'Brian Wiborg'
+maintainer_email  'bwiborg@chegg.com'
+license           'Apache 2.0'
+description       'Wrapper around chef_handler_sns containing IES specific patches.'
```
Comment (till) [inline ies-virtualbox/metadata.rb]: ;)

**Comment by till** [comment]
@baccenfutter do you mind picking the individual commits into smaller PRs? As in, pick the rvm removal into a PR, we can merge — you `git rebase master` this, and so on and so forth?

**Comment by till** [comment]
Generally 👍 , looks good, just try not to do make PRs with `diffs` no one can grok. ;))

**Comment by baccenfutter** [comment]
I got hung up while trying to write a test for ies-nodejs. Will skip that test tomorrow and move forward before re-iterating it again.

**Comment by baccenfutter** [comment]
This PR contains the missing scale-out of `sinopia`. I'll take it back, once I've returned.

**Comment by baccenfutter** [comment]
This PR concludes easybib/ops#213.

**Comment by baccenfutter** [comment]
@till, at least that is not how I understood the task. My understanding was to scale out the changes we've made to 3rd-party cookbooks. None of my PRs before my vacation actually changed the chef-run - most just removed stale cookbooks, some scaled out minor changes to `ies-younameit` cookbooks (like this one).

I think, actually changing the chef-run to use the new `ies-*` versions of the regarding cookbooks will be done in a separate second step.

I can of course go ahead and make the according changes to the chef run right now, while I am at it.
**Is this what we want?**

**Comment by baccenfutter** [comment]
Looking at it, I just realized that I forgot the tests for this PR. Taking it back for that.

File: ies-sinopia/spec/spec_helper.rb [hunk 2]
```diff
@@ -0,0 +1,6 @@
+require 'chefspec'
+
+RSpec.configure do |config|
+  config.platform = 'ubuntu'
+  config.version = '12.04'
```
Comment (fh) [inline ies-sinopia/spec/spec_helper.rb]: @baccenfutter This is not our ubuntu version, please update 14.04

Comment (baccenfutter) [inline ies-sinopia/spec/spec_helper.rb]: Good call! Copy-paste error on my behalf.

**Pull request merged**